src/main/webapp/jsp/adminSearch.jsp

```diff
@@ -1,555 +1,24 @@
-                                            %>
-
-
-
-                                            <li
-                                                class="list-group-item list-group-item-danger">
-                                                <strong> <%=data.feedbackSeesionLinkToNameMap
-                                    .get(link)%>
-                                            </strong> <input value=<%=link%>
-                                                readonly="readonly"
-                                                class="form-control"/ >
-                                            </li>
-
-
-
-                                            <%
-                                                }
-                                                        }
-                                            %>
-                                            
-                                            <%
-                                                if (data.studentPublishedFeedbackSessionLinksMap.get(student
-                                                                .getIdentificationString()) != null) {
-                                            %>
-
-
-
-                                            <%
-                                                for (String link : data.studentPublishedFeedbackSessionLinksMap
-                                                                    .get(student.getIdentificationString())) {
-                                            %>
-
-
-
-                                            <li
-                                                class="list-group-item list-group-item-success">
-                                                <strong> <%=data.feedbackSeesionLinkToNameMap.get(link)%>
-                                            </strong> <input value=<%=link%>
-                                                readonly="readonly"
-                                                class="form-control"/>
-                                            </li>
-
-
-
-                                            <%
-                                                }
-                                                        }
-                                            %>
-
-                                        </ul>
-                                    </td>
-
-                                </tr>
-                                <%
-                                    }
-
-                                    }
-                                %>
-                            </tbody>
-                        </table>
-                    </div>
-                </div>
-
-
-                <jsp:include page="<%=Const.ViewURIs.STATUS_MESSAGE%>" />
-
-            </div>
-
-
-
-    <jsp:include page="<%=Const.ViewURIs.FOOTER%>" />
-</body>
-</html>
\ No newline at end of file
+    <script type="text/javascript" src="/bootstrap/js/bootstrap.min.js"></script>
```

Comment:
@bamboo3250 apparently this is getting overwritten so if I don't include it, bootstrap doesn't work for the tooltip